src/Sylius/Component/Product/Model/ProductInterface.php

```diff
@@ -28,8 +29,7 @@
     SoftDeletableInterface,
     TimestampableInterface,
     ToggleableInterface,
-    ProductTranslationInterface,
-    AssociableInterface
```

Comment:
You should leave it here, because ProductInterface should force user to implement associations methods.